scripts/commands/build.js

```diff
@@ -44,12 +49,28 @@ async function buildRelease() {
 		return false;
 	}
 
-	await pullLatestChanges();
+	const git = gitFactory();
+	const currentBranchName = await getCurrentBranchName();
+
+	await git.checkout( 'master' );
+	await git.pull( [ 'origin', 'master', '--rebase' ] );
+
 	const version = getCurrentVersion();
 	info(
 		`Pulled latest changes from 'master'. Current version is ${ version }.`
 	);
 
+	const shouldContinue = await promptContinue(
+		'Would you like to create a new release?'
+	);
+
+	if ( ! shouldContinue ) {
+		info( 'Aborting release build.' );
+		await switchToBranchWithMessage( currentBranchName );
+
+		return false;
+	}
+
 	await execAsync( 'npm i' );
```

Comment:
If the user accepted to go forward it would be helpful to output a message telling the user that the process might take some time. It scared me at first that it wasn't responding and I `ctrl+c`d